great-idea-website/css/index.css

```diff
@@ -63,4 +63,89 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+nav {
+	padding: 20px;
+	margin-left: 50px;
+	color: grey;
+	margin: 20px;
+	word-spacing: 60px;
+}
+.tag .tagline-text {
+	display: inline-block;
+	padding: 10px 70px;
+	font-size: 80px;
+	text-align: center;
+	vertical-align: top;
+	margin-top: 120px;
+	font-family: 'Bangers', cursive;
+}
+.tag .tagline-text button {
+	font-family: 'Bangers', cursive;
+	font-size: 20px;
+	background-color: white;
+	margin-top: 5px;
+	padding: 8px 45px;
+	border: 1px solid black;
+	vertical-align: top;
+}
+.header-image {
+	display: inline-block;
+	margin-top: 70px;
+}
+.page {
+	border-top: 1px solid black;
+	padding-top: 40px;
+	margin-top: 75px;
+	margin-right: 300px;
+	margin-left: 20px;
+}
+.page .top-section {
+	padding: 0 20px;
+}
+.page .top-section .box {
+	display: inline-block;
+	width: 400px;
+	margin-right: 35px;
```

Comment:
if you add a margin-left on the .box  class, that would get the boxes to line up nicely in the top section.  I was playing a while with the margin : 0 auto, and I cannot figure out why the auto is not centering the two boxes in the top section.  Let me know if you find out why it's not centering.